cmds/exp/smbios_transfer/smbios_transfer.go

```diff
@@ -28,7 +29,7 @@ const (
 	maxWriteSize uint32 = 128
 
 	// IPMI blob ID on BMC
-	smbiosBlobID = "/smbios\x00"
+	smbiosBlobID = "/smbios"
```

Comment:
if you remove trailing null here, can you consider updating the comment as well, as BMC authentic ID would come with null

Reply:
Alternatively, leave this as is, and call TrimSuffix on both ends of the equal below

Reply:
Sounds good to me. I've updated the comment, because I think that's more clear and concise (as well as idiomatic/unsurprising).